kubernetes/linera-validator/templates/prometheus.yaml

```diff
@@ -22,16 +22,16 @@ kind: ServiceMonitor
 metadata:
   labels:
     release: linera-core
-  name: validator-1
+  name: proxy
```

Comment:
Curious what's happening here

Reply:
The proxy service in Kubernetes was originally named 'validator-1' as an oversight. It has subsequently been renamed to proxy.